### PR TITLE
New background and pulls spaces from DB.

### PR DESCRIPTION
**Pull request by kchason**
Removes the front end mapping, new background with equal squares to map to backend. Retrieves spaces from database via the model.

### CODE REVIEW

**Comment by JerroldJV** [comment]
I only see that the background image got changed.  Did you make sure you pushed all your commits to github?

**Comment by kchason** [comment]
Oops... I just pushed the rest

File: clueless/static/clueless/js/game.js [hunk 1]
```diff
@@ -1,33 +1,14 @@
...
-	[25, "Lounge-Dining Hallway", 0.76875 * canvas_width, 0.26875 * canvas_height, canvas_width, 0.3875 * canvas_height],
-	[26, "Library-Billiard Hallway", 0.2125 * canvas_width, 0.3875 * canvas_height, 0.325 * canvas_width, 0.60625 * canvas_height],
-	[27, "Billiard-Dining Hallway", 0.65625 * canvas_width, 0.3875 * canvas_height, 0.76875 * canvas_width, 0.60625 * canvas_height],
-	[28, "Library-Conservatory Hallway", 0, 0.60625 * canvas_height, 0.2125 * canvas_width, 0.71875 * canvas_height],
-	[29, "Billiard-Ball Hallway", 0.325 * canvas_width, 0.60625 * canvas_height, 0.65625 * canvas_width, 0.71875 * canvas_height],
-	[30, "Dining-Kitchen Hallway", 0.76875 * canvas_width, 0.60625 * canvas_height, canvas_width, 0.71875 * canvas_height],
-	[31, "Conservatory-Ball Hallway", 0.2125 * canvas_width, 0.71875 * canvas_height, 0.325 * canvas_width, canvas_height],
-	[32, "Ball-Kitchen Hallway", 0.65625 * canvas_width, 0.71875 * canvas_height, 0.76875 * canvas_width, canvas_height]  
-];
+//Get canvas object
```
Comment (rsfriedman) [inline clueless/static/clueless/js/game.js:1]: did we move all these values elsewhere?

Comment (kchason) [reply clueless/static/clueless/js/game.js:1]: No, we don't need them anymore. With the new plan, it's a 5x5 equal grid, so we can just take the canvas width and height and create the grid from there.

**Pull request merged**